deployability/modules/testing/tests/helpers/agent.py

```diff
@@ -311,25 +358,33 @@ def perform_uninstall_and_scan_for_agent(agent_params, wazuh_params) -> None:
 
 
     @staticmethod
-    def assert_results(result, agent_params) -> None:
+    def assert_results(result, params = None) -> None:
+
         """
         Gets the status of an agent given its name.
-        
+
         Args:
             result (dict): result of comparison between pre and post action scan
 
         """
-        os_type = HostInformation.get_os_type(agent_params)
+        os_type = HostInformation.get_os_type(params)
+
         if os_type == 'linux':
             categories = ['/root', '/usr/bin', '/usr/sbin', '/boot']
+
+        elif os_type == 'windows':
+            categories = ['C:\\Program Files', 'C:\\Program Files (x86)','C:\\Users\\vagrant']
+
         elif os_type == 'macos':
             categories = ['/usr/bin', '/usr/sbin']
+
         actions = ['added', 'modified', 'removed']
 
         # Testing the results
         for category in categories:
             for action in actions:
-                assert result[category][action] == [], logger.error(f'{result[category][action]} was found in: {category}{action}')
+
+                assert result[category][action] == [], logger.error(f'{result[category][action]} was found in: {category} {action}')
 
     def areAgent_processes_active(agent_params):
```

Comment:
- Change the name areAgent_processes_actto snake_case. 
- decorate this method as `@staticmethod`

Reply:
Done https://github.com/wazuh/wazuh-qa/pull/5310/commits/b1712ec86ca12fdc5c8469f100bdd7b487838276